scripts/setup.sh

```diff
@@ -12,3 +12,8 @@ fi
 if [[ ! -e .env.json ]]; then
     cp stickers.json.template stickers.json;
 fi
+
+# Initialize config.json
+if [[ ! -e .env.json ]]; then
+    cp config.json.template config.json;
+fi
```

Comment:
this will need to be executed manually on deployed instances bc running the setup will reset the db